docs/src/contributing/guide.md

```diff
@@ -1,6 +1,3 @@
----
```

Comment:
We need to keep these `yaml` headers, I think. Similarly for all other deletions below. 

Maybe @cpfiffer can confirm whether we need these.

Reply:
Yeah, these should all be left in.